app/views/cookie_preferences/show.html.erb

```diff
@@ -105,7 +98,7 @@
 
```

Comment:
Typo in line 102 - should be `relevancy` not `relevency` (I know this is in production already!)